### DISCUSSION TITLE
请问双语模式的作用是什么？

### DISCUSSION THREAD

**Discussion by MC-dusk** [diana7127/mpv.net-DW#21] [Q&A] [answered]
我mkv视频内挂了中英文两个srt字幕，启用了主次字幕，但是重叠了，开关双语模式都没有变化。
另外软件版本在哪看啊，我也不知道是不是最新版的，是不是已经修了

**Comment by diana7127** [comment, marked as answer]
1. 按照[使用说明](https://github.com/diana7127/mpv.net-DW#%E5%8F%B3%E9%94%AE%E8%8F%9C%E5%8D%95-1)顺序操作；或开启双语模式后，关闭字幕再重新打开字幕
2. 模组版本可在`右键 > 帮助 > 关于`查看，DW版本号目前无法查看，可根据更新内容比较判断

**Comment by MC-dusk** [comment]
好的，双语模式问题解决了，版本如果能在文件夹里面加一个DW版本的记号就好了，比如一个version.txt

**Comment by MC-dusk** [comment]
抱歉又发现一个问题，如果视频只有一个字幕，就不能用快捷键w开关（切换）字幕，需要手动到track列表选择字幕，如果有两个或以上的字幕就可以直接用快捷键w切换各个字幕或者切换到空轨道就是关闭字幕了

另外双语字幕，如果主字幕的轨道落后于次字幕，用快捷键w最多只能前进到次字幕的轨道，如果已经超过次字幕的轨道则不受影响，直到循环到下一轮。

无伤大雅的问题，就是挺奇怪的。

**Reply by MC-dusk** [reply]
偶然发现，虽然单字幕情况下切换主字幕（我定义的快捷键w）失效了，但切换副字幕（默认快捷键e）仍然生效，所以第一个问题算是半解决了。
